### ISSUE TITLE
medicaid policy data

### ISSUE THREAD

**Issue by paghosh** [paghosh/medicaid_pregnant_women#1]
Hi Ahmed,

I have added some state-level medicaid policy data in the data folder in Github. Could you please make a state-level panel data set by combing all the excel files (x1-x5 medicaid policy variables and state-level control variables) for this week's RA work?

Best,
Pallab

**Comment by ahmedelfatmaoui** [comment]
Hi Dr. Ghosh,

Can we have the RA do the following for  "x1_Medicaid eligibility FLP% 2006-2022.xlsx":

- keep only common variables in all the sheets: FY2006 to FY2022 (create another excel for those that don't exist consistently)
- use the same variable or column names in each sheet to refer to the same variables

**Comment by paghosh** [comment]
Yes. It makes sense. Please do so and we can discuss about it next week.

**Comment by ahmedelfatmaoui** [comment]
Good data collection practices for the RA:

- Please use the same measurement for each column;  for instance, in the following use the highlighted format (decimal) or the other one (percentage) but don't mix them:

<img width="162" alt="image" src="https://github.com/paghosh/medicaid_pregnant_women/assets/89555208/c4788e5a-46a0-42b3-8be3-646f0db39c22">

**Comment by ahmedelfatmaoui** [comment]
Just for future reference: 

- [The code used to concatenate the data ](https://github.com/paghosh/medicaid_pregnant_women/blob/c8997309b096f899fc8882d6904584ef01a8f8e8/scr/data-cleaning/create-panel.R)
- [The combined data](https://github.com/paghosh/medicaid_pregnant_women/tree/c8997309b096f899fc8882d6904584ef01a8f8e8/data/output)
